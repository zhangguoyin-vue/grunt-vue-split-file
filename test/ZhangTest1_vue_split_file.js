
    const zhangTest1 = new Promise((resolve, reject) => {
        require([
            './src/commHelp.js',
            'json!./src/zhengJson.json',
        ], function(commHelp, zhangJson) {
            resolve({
                name: 'ZhangTest1',
                components: {
                    HelloWorld:vueLoad('./HelloWorld.vue'),
                },
                data:function() {
                    return {
                        msg: "Welcome to Your Vue.js App",
                        pagePara: "",
                        isCollapse: false
                    };
                },
                computed: {},
                created: function() {
                    this.pagePara = zhangJson[0].name + commHelp.getColorByIndex(1);
                },
                watch: {},
                mounted() {},
                methods: {}
            });
        });
    });
    module.exports = zhangTest1;
