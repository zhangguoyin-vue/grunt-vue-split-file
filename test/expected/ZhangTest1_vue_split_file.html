<template>
    <div>
        <div>{{pagePara}}</div>
        <div>
            <hello-world></hello-world>
        </div>
    </div>
</template>